beta/kustomize/overlays/README.md

```diff
@@ -0,0 +1,12 @@
+# Overlays
+
+Contained here are overlays to customize and configure 1Password SCIM Bridge.
+
+`letsencrypt` contains the patches necessary to use the built in Let's Encrypt integration
```

Comment:
```suggestion
`letsencrypt` contains the patches necessary to use the default option of utilizing Let's Encrypt for your certificate management of your SCIM bridge
```

Wording suggestion to highlight that Let's Encrypt is the default TLS management option.

Reply:
resolved in upcoming commit.